tasks/main.yml

```diff
@@ -157,37 +165,98 @@
     mode=0755
   become_user: "{{ project_user }}"
 
+- name: configure celery and celery beat
+  template: src=celery.conf
+            dest=/etc/supervisor/conf.d/{{ project_name }}-celery-{{ item.name }}.conf
+            owner=root
+            group=root
+            mode=0600
+  with_items:
+    - { name: "default", command: "worker", flags: "--loglevel=INFO" }
+    - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile={{ root_dir }}/celerybeat.pid --loglevel=INFO" }
+  notify:
+    - restart celery
+    - restart celery-beat
+
+- name: ensure celery is running
+  supervisorctl: name={{ project_name }}-celery-default state=started
```

Comment:
Well, I manually reloaded supervisor and ran the deploy again and got a different error: `TASK [tequila-django : ensure celery is running] *******************************
fatal: [server1]: FAILED! => {"changed": false, "failed": true, "msg": "our_project-celery-default: ERROR (no such file)\n"}
`

I'm pretty sure that's just because this project doesn't use celery and so doesn't install it. But it does suggest that (if we are using celery) that reloading supervisor before trying this task will get past the previous error.